### PR TITLE
N°5786 - Problem with text color in public log and in AttributeHTML

### PR DESCRIPTION
**Pull request by accognet**
Internal

### CODE REVIEW

File: css/backoffice/vendors/_bulma-variables-overload.scss [hunk 1]
```diff
@@ -13,6 +13,9 @@ $body-overflow-x: hidden !default;
 $body-overflow-y: auto !default;
 
 
+  /*N°5786 - Problem with text color in public log and in AttributeHTML - override bulma css */
+$text-strong: inherit !default;;
```
Comment (Molkobain) [inline css/backoffice/vendors/_bulma-variables-overload.scss]:
```suggestion
/*N°5786 - Avoid strong text to always be grey (default Bulma color for this var.) so strong text can keep its color. This  is mostly for text within the .ibo-is-html-content. */
$text-strong: inherit !default;
```

**Review by Molkobain — CHANGES_REQUESTED** [review]

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
@steffunky the drawback is that it applies to more than the .ibo-is-html-content, but as we don't use bulma classes much, I felt it was okay. What do you think?

**Comment by steffunky** [comment]
Doest it affect Alerts block ?
Or Backoffice communications' alerts ?

**Comment by accognet** [comment]
Yes, it solves the same problem

**Comment by piRGoif** [comment]
Technical review : OK

**Comment by piRGoif** [comment]
Functional review : OK

**Pull request merged**